support-frontend/assets/pages/[countryGroupId]/checkout/helpers/backButton.ts

```diff
@@ -0,0 +1,15 @@
+import type { ProductDescription } from 'helpers/productCatalog';
+import { routes } from 'helpers/urls/routes';
+
+const validBackButtonPathOverrides: Record<string, string | undefined> = {
+	subscriptionsLanding: routes.subscriptionsLanding,
```

Comment:
Is it worth labelling these `subscribe` and possibly add `contribute` too?

Reply:
Yeah happy to change to subscribe, it's nice and short. I was thinking of holding off adding any more overrides until we need them so will leave it at that for now if that sounds okay?